packages/firestore-compat/karma.conf.js

```diff
@@ -29,12 +31,20 @@ module.exports = function (config) {
 
     // frameworks to use
     // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
-    frameworks: ['mocha'],
+    frameworks: ['mocha']
+  };
 
-    client: Object.assign({}, karmaBase.client, {
-      firestoreSettings: getFirestoreSettings(argv)
-    })
-  });
+  if (argv.targetBackend) {
+    karmaConfig.client = {
+      ...karmaConfig.client,
+      targetBackend: argv.targetBackend
+    };
+  } else if (argv.local) {
```

Comment:
can you add a comment about why the `else if` is needed? is it because some existing CI still uses `argv.local`? Can we update those to set `argv.targetBackend` to `emulator` ?

Reply:
I see, Thanks